drivers/pms5003/include/pms5003_constants.h

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright (C) 2021 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_pms5003
+ *
+ * @{
+ * @file
+ * @brief       Constants for PMS5003
+ *
+ * @author      Francisco Molina <francois-xavier.molina@inria.fr>
+ */
+
+#ifndef PMS5003_CONSTANTS_H
+#define PMS5003_CONSTANTS_H
+
+#include "pms5003.h"
+#include "timex.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief   Symbol rate of the PMS5003 UART interface
+ */
+#define PMS5003_BAUDRATE            (9600)
+/**
+ * @brief   PMS5003 uart command length
+ *
+ *          2 + 2 + 1 + 2 (start + length + cmd + check bytes)
+ */
+#define PMS5003_CMD_FRAME_LEN       7U
+/**
+ * @brief   PMS5003 particualate matter measurements frame length
+ *
+ *          2 + 2x13 + 2 (start + length + data + check bytes)
+ */
+#define PMS5003_PM_FRAME_LEN        32U
+/**
+ * @brief   PMS5003 non-read cmd response frame length
+ *
+ *          2 + 2 + 2 + 2 (start + length + data + check bytes)
+ */
+#define PMS5003_RESP_FRAME_LEN      8U
+/**
+ * @brief   PMS5003 frame crc length, 2 bytes
+ */
+#define PMS5003_CRC_LEN             2U
```

Comment:
The checksum is indeed only a sum, not a CRC. You could use the generic term frame check sequence (FCS) instead, if you dislike checksum.